src/cmp_aus.js

```diff
@@ -1,57 +1,20 @@
 const { URL } = require('url');
 const synthetics = require('Synthetics');
-const logger = require('SyntheticsLogger');
+const {
+	log,
+	logError,
+	clearCookies,
+	clearLocalStorage,
+	checkTopAdHasLoaded,
+	checkCMPIsOnPage,
+	checkCMPIsNotVisible,
+	loadPage,
+	reloadPage,
+} = require('./utils');
 
 const LOG_EVERY_REQUEST = false;
 const LOG_EVERY_RESPONSE = false;
 
-let startTime = new Date().getTime();
-const getTimeSinceStart = () => new Date().getTime() - startTime;
```

Comment:
I personally don't see the benefit in having the timestamps in the log messages so I've deleted them, but happy to reinstate if anyone feels strongly about it!